src/main/resources/resource/WebGui/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{

```

Comment:
is this intended to be checked in?

Reply:
yes-ish  .... it specifies how debugging could proceed in vscode - something a newbie would not want to figure out